setup.py

```diff
@@ -75,6 +75,11 @@ def package_files(directory, strip_leading):
           ],
           'ci': ['codecov'],
           'tf': ['tensorflow>=2.2.0'],
+          'torch': [
```

Comment:
Formatting seems off?

Reply:
Just fixed. Sorry about that. I'll need to get VSCode to format on save